docs/storage/storage.md

```diff
@@ -208,6 +209,44 @@ Optionally, you can specify Azure folder structure name in the file **zeppelin-s
 ```
 
 </br>
+
+## Notebook Storage in Apache Hadoop HDFS <a name="HDFS"></a>
+
+Notebooks may be stored in HDFS and local file system.
+
+To use this, set the following environment variable in the file **zeppelin-site.xml**
+
+```
+<property>
+  <name>zeppelin.notebook.storage</name>
+  <value>org.apache.zeppelin.notebook.repo.HDFSNotebookRepo</value>
+  <description>notebook persistence layer implementation</description>
+</property>
+
+<property>
+  <name>hdfs.url</name>
```

Comment:
I think it is better to add prefix zeppelin.notebook, because hdfs is used in many places. Adding prefix can help user to understand this this is for notebook storage.